pkg/client/observatorium/observability_config.go

```diff
@@ -123,6 +124,12 @@ type ObservabilityCloudwatchLoggingConfigCredentials struct {
 	SecretAccessKey string `yaml:"aws_secret_access_key" validate:"required"`
 }
 
+type ObservabilityEnterpriseCloudwatchLoggingConfigCredentials struct {
+	AccessKey       string `yaml:"aws_access_key" validate:"-"`
+	SecretAccessKey string `yaml:"aws_secret_access_key" validate:"-"`
+	OrgId           string `yaml:"org_id" validate:"-"`
```

Comment:
Isn't the OrgId required? can we add validation for it?

Reply:
:+1: